apps/wallet/src/ui/app/shared/collapse/index.tsx

```diff
@@ -11,15 +11,37 @@ interface CollapsibleProps {
 	defaultOpen?: boolean;
 	children: ReactNode | ReactNode[];
 	shade?: 'lighter' | 'darker';
+	isExternalControlled?: boolean;
```

Comment:
I think you can get this to work without the `isExternalControlled` prop, I had to do something very similar for https://github.com/MystenLabs/sui/blob/1227ee1cee3f81d2b1f0ef889a5e32452824fd81/sdk/dapp-kit/src/components/connect-modal/ConnectModal.tsx#L24

Reply:
Cool that works. Thx!